boards/px4/fmu-v4/src/init.c

```diff
@@ -399,5 +399,15 @@ __EXPORT int board_app_initialize(uintptr_t arg)
 
 #endif
 
+	/*
```

Comment:
I think we are not doing the GPIO init  and this just masks the root issues.

As a side note are we ok delaying Boot? We can Break the reset into 2 pieces, it is all GPIO manipulation. Make the arch init set things up correctly.  Then use the delay 3+ ms.  from the aching to the app init.  Then set up a call back to finish the job.

Reply:
Yes I figured this would prompt the discussion to check for what's actually needed.